server/src/main/resources/routes

```diff
@@ -12,8 +12,9 @@ GET           /health-check/received-landing-times/:from/:to/:minimumToConsider
 GET           /health-check/received-arrival-updates/:from/:to/:minimumToConsider/:lastUpdatedMinutes  controllers.application.HealthCheckController.receivedUpdates(from: String, to: String, minimumToConsider: Int, lastUpdatedMinutes: Int)
 GET           /health-check/calculated-desk-updates                                   controllers.application.HealthCheckController.deskUpdates()
 
-GET           /airport-config                                                         controllers.application.ConfigController.getAirportConfig
-GET           /pax-feed-source-order                                                  controllers.application.ConfigController.getPaxFeedSourceOrder
+GET           /config/airport-config                                                  controllers.application.ConfigController.getAirportConfig
```

Comment:
e2e/cypress/e2e/api-role-access.cy.ts:12 seems to using /airport-config and e2e are passing . But if want it e2e consistent with api then can update the file.